packages/myst-config/src/site/types.ts

```diff
@@ -23,33 +23,19 @@ export interface SiteAction extends SiteNavPage {
   static?: boolean;
 }
 
-export interface SiteAnalytics {
-  google?: string;
-  plausible?: string;
-}
-
 export const SITE_CONFIG_KEYS = {
-  optional: [
-    'projects',
-    'nav',
-    'actions',
-    'domains',
-    'twitter',
-    'logo',
-    'logo_text',
-    'favicon',
-    'analytics',
-  ].concat(SITE_FRONTMATTER_KEYS),
```

Comment:
Do these getting removed effect the downstream stuff right away?

Reply:
The effects of removing these could be:
- Validation errors - I think these will not happen because site config now allows any arbitrary key/value template options
- Fields getting ignored in site build - Need to be a little careful here. Logo, logo_text, twitter can all be added as template options, so those should continue to work as before once everything is said and done. Analytics is a structured field so we may need some backwards compatibility massaging to pull that into template options... (`analytics.google` -> `analytics_google`) or whatever.

Reply:
Nice. Ok, thanks for the explanation.